apps/src/templates/teacherNavigation/LessonMaterialsContainer.tsx

```diff
@@ -103,17 +108,26 @@ const LessonMaterialsContainer: React.FC = () => {
 
   return (
     <div>
-      <SimpleDropdown
-        labelText={i18n.chooseLesson()}
-        isLabelVisible={false}
-        onChange={event => onDropdownChange(event.target.value)}
-        items={lessonOptions}
-        selectedValue={
-          selectedLesson ? selectedLesson.id.toString() : 'no lesson'
-        }
-        name={'lessons-in-assigned-unit-dropdown'}
-        size="s"
-      />
+      <div className={styles.lessonMaterialsPageHeader}>
+        <SimpleDropdown
+          labelText={i18n.chooseLesson()}
+          isLabelVisible={false}
+          onChange={event => onDropdownChange(event.target.value)}
+          items={lessonOptions}
+          selectedValue={
+            selectedLesson ? selectedLesson.id.toString() : 'no lesson'
```

Comment:
Is the 'no lesson' text ever shown or should we just use ''?

Reply:
I switched to '' - realistically, if there is a no lesson, no dropdown should show up.